sdk/core/Azure.Core.Experimental/src/Variant/Variant.Null.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Azure
+{
+    public readonly partial struct Variant
+    {
+        /// <summary>
+        /// Null Variant.
+        /// </summary>
+        public static readonly Variant Null = new((object?)null);
+
+        /// <summary>
+        /// Indicates whether the Variant is null or has a value.
+        /// </summary>
+        /// <returns><code>true</code> if the Variant is <code>null</code>; <code>false</code> otherwise.</returns>
+        public static bool IsNull(Variant value) => value.Type == null;
```

Comment:
why is this a static method and not an instance property?

Reply:
I was thinking of mirroring string.IsNullOrEmpty(), but if you like it better as an instance method/property, we can totally do that (I actually wrote it that way first and changed my mind).